src/xb-self-test.c

```diff
@@ -2641,6 +2659,7 @@ main (int argc, char **argv)
 	g_test_add_func ("/libxmlb/xpath-node", xb_xpath_node_func);
 	g_test_add_func ("/libxmlb/xpath-parent-subnode", xb_xpath_parent_subnode_func);
 	g_test_add_func ("/libxmlb/multiple-roots", xb_builder_multiple_roots_func);
+	g_test_add_func ("/libxmlb/require-root", xb_builder_require_root_func);
```

Comment:
```suggestion
	g_test_add_func ("/libxmlb/single-root", xb_builder_single_root_func);
```

aiui that name would better reflect what’s being tested